### ISSUE TITLE
Code not being executed?

### ISSUE THREAD

**Issue by huachuman** [SilentVoid13/Templater#1092]
**Plugin information (please complete the following information):**

-   OS: Windows 11
-   Templater version: latest
-   Obsidian version: latest
-   Templater settings: "Templates" folder (default?)

**Describe the bug**
It's not executing code in notes. I put the following inside a template but when creating a new note based on that template, it doesn't rename the note.

<%*
	let title = "SomeTitle";
	await tp.file.rename(`${title}`);
_%>

It also does not execute code anywhere, like 

[[<% tp.date.now("YYYY-MM-DD") %>]]

Just renders as [[<% tp.date.now("YYYY-MM-DD") %>]]

**Expected behavior**
The code works

**Screenshots**

**Additional context**
I made sure to enable the execute command on new file creation option.

**Comment by Zachatoo** [comment]
How are you creating the note? If you're not creating notes using the Templater command `Create new note from template`, then the template code will not be executed. The template code is executed when running a Templater command, it does not run automatically.

If you want the template code to be executed when creating files without using a Templater command, you can setup folder templates.

1. Turn on "Trigger Templater on new file creation".
2. Turn on "Enable folder templates".
3. Add a folder template. Any notes created in the provided folder will have the provided template applied to it.

If you want a template to be applied to all new notes, regardless of folder, you can setup a folder template with `/` as the folder.

**Comment by Zachatoo** [comment]
I agree, it's not obvious to the average user how to use this plugin. That's why people like me like to help in here and on the Obsidian Discord.

I disagree that folder templates should be a more prominent feature. I'd imagine most people are applying their templates manually using a Templater command. At least that's what I've seen in my activities on the Obsidian Discord.

I didn't make any video, I'm just a random stranger helping out, I am not the maintainer of this plugin.

Feedback is appreciated, hopefully at some point we can make this less confusing for future users within the plugin or documentation.

**Comment by NishantDate** [comment]
Hey this is probably pretty late but I'm getting a bug where if I 
1. Enable "Trigger templater on new file creatoin"
2. Enable Folder templates
3. Then Try to create a new note with a template like {{date}}

I only get {{date}} shown instead of the actual date. i.E templater doesn't execute. 

However, when using periodic notes/daily notes my daily note template executes. Any idea why this would be the case?

**Comment by Zachatoo** [comment]
Because `{{date}}` is not Templater syntax, it's periodic notes syntax.

You need to use [tp.date.now](https://silentvoid13.github.io/Templater/internal-functions/internal-modules/date-module.html#tpdatenowformat-string--yyyy-mm-dd-offset-numberstring-reference-string-reference_format-string) instead.

**Comment by NishantDate** [comment]
Thank you for the quick response! Is there templator docs for "time" as well?

**Comment by Zachatoo** [comment]
You'll also use `tp.date.now()` for times. You'll just pass a different format, for example `hh:mm`. The docs have a link to the moment docs for all the format tokens you have access to.

**Comment by NishantDate** [comment]
I was blind. I see them now. Thank you again!
